manage.py

```diff
@@ -808,6 +808,55 @@ def lint(filename, additional_properties):
     write_yaml_file(filename, elements)
 
 
+@cli.command()
+def update_sustainability_elements():
+    """Update mapping/sustainability.yaml"""
```

Comment:
How does this interact with `update_sustainability_docs`?

Reply:
This function updates the CoST IDS elements in `sustainability.yaml`, using the spreadsheet in which CoST maintain the sustainability elements as an input, whilst `update_sustainability_docs` updates the Markdown mapping documentation, using `sustainability.yaml` as an input.

I've updated the docstrings in https://github.com/open-contracting/infrastructure/pull/425/commits/6b1a950834b09d16e0cefcdf643b43ad164b5105